### PR TITLE
LPS-55518 Use EditorConfig to provide configuration for editors so it can be extended and configured

### PR DESCRIPTION
**Pull request by sergiogonzalez**

### CODE REVIEW

**Comment by sergiogonzalez** [comment]
cc/ @jbalsas  @ipeychev @ambrinchaudhary @csierra

**Comment by brianchandotcom** [comment]
Rebase error :( please resend? Thx. @sergiogonzalez

File: modules/frontend/frontend-editors-web/src/META-INF/resources/html/editors/ckeditor_diffs/extension/creole_dialog_definition.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+				infoTab.remove('vAlign');
+				infoTab.remove('width');
+				infoTab.remove('widthType');
+				infoTab.remove('wordWrap');
+
+				dialogDefinition.minHeight = 40;
+				dialogDefinition.minWidth = 210;
+			}
+			else if (dialogName === 'table' || dialogName === 'tableProperties') {
+				debugger;
```
Comment (natecavanaugh) [inline modules/frontend/frontend-editors-web/src/META-INF/resources/html/editors/ckeditor_diffs/extension/creole_dialog_definition.js]: Hey guys, why is there a debugger call in here?

Comment (jbalsas) [inline modules/frontend/frontend-editors-web/src/META-INF/resources/html/editors/ckeditor_diffs/extension/creole_dialog_definition.js]: So sorry, Nate, I missed it :disappointed: 

See https://github.com/brianchandotcom/liferay-portal/pull/27501

**Pull request closed without merging**